analysis/dataflow/pointer.go

```diff
@@ -37,14 +38,20 @@ import (
 //
 // If error != nil, the *pointer.Result is such that every Value in the functions f such that functionFilter(f) is true
 // will be in the Queries or IndirectQueries of the pointer.Result
-func DoPointerAnalysis(p *ssa.Program, functionFilter func(*ssa.Function) bool, functionSet map[*ssa.Function]bool) (*pointer.Result,
-	error) {
+func DoPointerAnalysis(c *config.Config, p *ssa.Program,
```

Comment:
nit: `c` could have type `*config.PointerConfig` since it doesn't use all the fields of `*config.Config`

Reply:
It could but I think having only one type of config throughout just makes things a little easier to read and refactor (if in the future we want to share more config options between our config and the pointer's config, e.g. log-level).
(Which also reminds me we were talking about using slog at some point)